app/controllers/employees_controller.rb

```diff
@@ -11,27 +11,33 @@ def new
   end
 
   def show
-
+    @employees = Employee.find(params[:id])
```

Comment:
you can use a hook, in this case before_action, you could create a private method called set_employee like this:
```ruby
def set_employee
  @employee = Employee.find(params[:id])
end
```